src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -5001,7 +5009,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
   <Target
     Name="_GetCopyToOutputDirectoryItemsFromTransitiveProjectReferences"
     DependsOnTargets="_PopulateCommonStateForGetCopyToOutputDirectoryItems;_AddOutputPathToGlobalPropertiesToRemove"
-    Returns="@(_TransitiveItemsToCopyToOutputDirectory)">
+    Returns="@(_CopyToOutputDirectoryTransitiveItems)">
```

Comment:
I am changing the name from _TransitiveItemsToCopyToOutputDirectory to _CopyToOutputDirectoryTransitiveItems b/c otherwise _TransitiveItemsToCopyToOutputDirectory items are duplicated. They get added in this target and the caller then adds the returned items into the same item group.